### ISSUE TITLE
Lodash-es causing Vite build issue in 0.6.3

### ISSUE THREAD

**Issue by Keithcat767** [fi3ework/vite-plugin-checker#302]
### Describe the bug

After updating to 0.6.3 we are seeing the following error:

```failed to load config from /Users/keith/Desktop/projects/ui_alpha/vite.config.js
error during build:
Error [ERR_REQUIRE_ESM]: require() of ES Module node_modules/lodash-es/lodash.js from node_modules/vite-plugin-checker/dist/cjs/main.js not supported.
Instead change the require of lodash.js in node_modules/vite-plugin-checker/dist/cjs/main.js to a dynamic import() which is available in all CommonJS modules.```

### Reproduction

1. Install version 0.6.3
2. Run `vite build`
3. Observe the error

### Expected behavior

Vite build will work with this version

### System Info

```shell
System:
    OS: macOS 13.4
    CPU: (8) arm64 Apple M2
    Memory: 75.06 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.12.0 - ~/.nvm/versions/node/v18.12.0/bin/node
    Yarn: 1.22.21 - ~/.nvm/versions/node/v18.12.0/bin/yarn
    npm: 9.9.0 - ~/Desktop/projects/ui_alpha/node_modules/.bin/npm
    bun: 1.0.7 - ~/.bun/bin/bun
  Browsers:
    Chrome: 121.0.6167.85
    Safari: 16.5
  npmPackages:
    vite-plugin-checker: ^0.6.3 => 0.6.3
```

### Validations

- [X] Read the [docs](https://github.com/fi3ework/vite-plugin-checker#readme).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by buqiyuan** [comment]
I have the same problem, and here's my temporary solution: 
```
 "pnpm": {
    "overrides": {
      "vite-plugin-checker>lodash-es": "npm:lodash"
    }
  }
```

**Comment by Artur-** [comment]
This is fixed by #303 in 0.6.4

**Comment by Keithcat767** [comment]
Works great! Thanks @nclavaud and @Artur-
